tests/sqlite/relationships/test_sqlite_relationships.py

```diff
@@ -49,6 +49,28 @@ def get_is_admin(self):
         return "You are an admin"
 
 
+class Store(Model):
+
+    __connection__ = "dev"
+
+    @belongs_to_many("store_id", "product_id", "id", "id")
+    def products(self):
+        return Product
+
+    @belongs_to_many("store_id", "product_id", "id", "id", table="product_table")
```

Comment:
@josephmancuso I am not sure to understand why this `product_table` should be specified and also the way it is specified (through a relationship and not just an attribute) 🤔